python/example/ou_lif/ou_lif.py

```diff
@@ -0,0 +1,342 @@
+    c_mem = C_mem / area_m2
+    # reversal potential in mV
+    V_rev = -65.0
+    # spiking threshold in mV
+    V_th = -55.0
+    # initial synaptic weight in nC
+    h_0 = 4.20075
+    # leak resistance in MOhm
+    R_leak = 10.0
+    # membrane time constant in ms
+    tau_mem = 2.0
+
+    lif = arb.mechanism("lif")
+    lif.set("R_leak", R_leak)
+    lif.set("R_reset", 1e-10)
+    # set to initial value to zero (background input is applied via stochastic ou_bg_mech)
+    lif.set("I_0", 0)
+    lif.set("i_factor", i_factor)
+    lif.set("V_rev", V_rev)
+    lif.set("V_reset", -70.0)
+    lif.set("V_th", V_th)
+    # refractory time in ms
+    lif.set("t_ref", tau_mem)
+
+    # stochastic background input current (point mechanism)
+    # =====================================================
+
+    # synaptic time constant in ms
+    tau_syn = 5.0
+    # mean in nA
+    mu_bg = 0.15
+    # volatility in nA
+    sigma_bg = 0.5
+    # instantiate mechanism
+    ou_bg = arb.mechanism("noisy_expsyn_curr")
+    ou_bg.set("mu", mu_bg)
+    ou_bg.set("sigma", sigma_bg)
+    ou_bg.set("tau", tau_syn)
+
+    # stochastic stimulus input current (point mechanism)
+    # ===================================================
+
+    # number of neurons in the putative population
+    N = 25
+    # firing rate of the putative neurons in Hz
+    f = 100
+    # synaptic weight in nC
+    w_out = h_0 / R_leak
+    # mean in nA
+    mu_stim = N * f * w_out
+    # volatility in nA
+    sigma_stim = np.sqrt((1000.0 * N * f) / (2 * tau_syn)) * w_out
+    # instantiate mechanism
+    ou_stim = arb.mechanism("noisy_expsyn_curr")
+    ou_stim.set("mu", mu_stim)
+    ou_stim.set("sigma", sigma_stim)
+    ou_stim.set("tau", tau_syn)
+
+    # paint and place mechanisms
+    # ==========================
+
+    decor = arb.decor()
+    decor.set_property(Vm=V_rev, cm=c_mem)
+    decor.paint("(all)", arb.density(lif))
+    decor.place('"center"', arb.synapse(ou_stim), "ou_stim")
+    decor.place('"center"', arb.synapse(ou_bg), "ou_bg")
+    decor.place('"center"', arb.threshold_detector(V_th), "spike_detector")
+
+    return arb.cable_cell(tree, decor, labels)
+
+
+class ou_recipe(arb.recipe):
+    def __init__(self, cell, cat):
+        arb.recipe.__init__(self)
+
+        # simulation runtime parameters in ms
+        self.runtime = 20000
+        self.dt = 0.2
+
+        # initialize catalogue and cell properties
+        self.the_props = arb.neuron_cable_properties()
+        self.the_props.catalogue = cat
+        self.the_cell = cell
+
+        # protocol for stimulation
+        # "time_start": starting time, "scheme": protocol type
+        self.bg_prot = {"time_start": 0, "scheme": "FULL", "label": "ou_bg"}
+        self.stim1_prot = {"time_start": 10000, "scheme": "TRIPLET", "label": "ou_stim"}
+        self.stim2_prot = {
+            "time_start": 15000,
+            "scheme": "ONEPULSE",
+            "label": "ou_stim",
+        }
```

Comment:
```suggestion
        self.stim2_prot = {"time_start": 15000, "scheme": "ONEPULSE", "label": "ou_stim"}
```
Formatting adapted

Reply:
Our linter complains when changing this to a one-liner, so I'll leave it as is.